docs/StateRestore-action.md

```diff
@@ -0,0 +1,26 @@
+# State Restore
+
+ Description
+ -----------
+ State restore fetches previously store state from a tracking table and sets a run-time argument to be used for rest of the pipeline. The state is stored in a variable called ${state} which can
```

Comment:
`previously store state` => `previously stored state`

`The state is stored in a variable called ${state}` => `The state fetched is assigned to a variable called ${state}`